### ISSUE TITLE
Merge conflicts with latest baystation12 master branch

### ISSUE THREAD

**Issue by lmas** [Apollo-Community/ApolloStation#80]
```
git pull upstream master
git diff --name-only --diff-filter=U
```

Gave me a humoungus list... Got lot's of work for ya guys.
- [x] .gitignore
- [x] baystation12.dme
- [x] code/controllers/master_controller.dm
- [x] code/defines/obj/weapon.dm
- [x] code/game/area/Space Station 13 areas.dm
- [x] code/game/gamemodes/events/biomass.dm
- [x] code/game/gamemodes/game_mode.dm
- [x] code/game/jobs/jobs.dm
- [x] code/game/machinery/camera/camera.dm
- [x] code/game/machinery/camera/tracking.dm
- [x] code/game/machinery/doors/blast_door.dm
- [x] code/game/machinery/jukebox.dm
- [x] code/game/machinery/kitchen/processor.dm
- [x] code/game/machinery/machinery.dm
- [x] code/game/machinery/vending.dm
- [x] code/game/objects/effects/biomass_rift.dm
- [x] code/game/objects/effects/glowshroom.dm
- [x] code/game/objects/items/weapons/handcuffs.dm
- [x] code/game/objects/items/weapons/manuals.dm
- [x] code/game/objects/items/weapons/storage/uplink_kits.dm
- [x] code/game/objects/items/weapons/tools.dm
- [x] code/game/objects/items/weapons/twohanded.dm
- [x] code/game/objects/random/random.dm
- [x] code/game/objects/structures/girders.dm
- [x] code/game/turfs/simulated/walls.dm
- [x] code/global.dm
- [x] code/modules/admin/topic.dm
- [x] code/modules/clothing/clothing.dm
- [x] code/modules/clothing/head/misc_special.dm
- [x] code/modules/clothing/suits/miscellaneous.dm
- [x] code/modules/clothing/under/miscellaneous.dm
- [x] code/modules/customitems/item_defines.dm
- [x] code/modules/hydroponics/grown_inedible.dm
- [x] code/modules/hydroponics/trays/tray.dm
- [x] code/modules/hydroponics/vines.dm
- [x] code/modules/mob/living/carbon/carbon.dm
- [x] code/modules/mob/living/carbon/give.dm
- [x] code/modules/mob/living/carbon/human/examine.dm
- [x] code/modules/mob/living/silicon/robot/robot_modules.dm
- [x] code/modules/mob/mob.dm
- [x] code/modules/mob/mob_grab.dm
- [x] code/modules/paperwork/pen.dm
- [x] code/modules/power/port_gen.dm
- [x] code/modules/projectiles/ammunition/boxes.dm
- [x] code/modules/projectiles/ammunition/bullets.dm
- [x] code/modules/projectiles/gun.dm
- [x] code/modules/projectiles/guns/energy/laser.dm
- [x] code/modules/projectiles/guns/energy/stun.dm
- [x] code/modules/projectiles/guns/launcher/crossbow.dm
- [x] code/modules/projectiles/guns/projectile.dm
- [x] code/modules/projectiles/guns/projectile/automatic.dm
- [x] code/modules/projectiles/guns/projectile/pistol.dm
- [x] code/modules/projectiles/guns/projectile/revolver.dm
- [x] code/modules/projectiles/guns/projectile/shotgun.dm
- [x] code/modules/projectiles/projectile.dm
- [x] code/modules/projectiles/projectile/bullets.dm
- [x] code/modules/reagents/Chemistry-Machinery.dm
- [x] code/modules/reagents/dartgun.dm
- [x] code/modules/reagents/reagent_containers/food/snacks.dm
- [x] code/modules/reagents/reagent_containers/syringes.dm
- [x] code/modules/reagents/reagent_dispenser.dm
- [x] code/modules/reagents/syringe_gun.dm
- [x] code/modules/virus2/items_devices.dm
- [x] code/setup.dm
- [ ] icons/mob/AI.dmi
- [ ] icons/mob/back.dmi
- [ ] icons/mob/head.dmi
- [ ] icons/mob/items_lefthand.dmi
- [ ] icons/mob/items_righthand.dmi
- [ ] icons/mob/mask.dmi
- [ ] icons/mob/uniform.dmi
- [ ] icons/obj/ammo.dmi
- [ ] icons/obj/clothing/hats.dmi
- [ ] icons/obj/clothing/uniforms.dmi
- [ ] icons/obj/custom_items.dmi
- [ ] icons/obj/food.dmi
- [ ] icons/obj/gun.dmi
- [ ] icons/obj/harvest.dmi
- [ ] icons/obj/items.dmi
- [ ] icons/obj/projectiles.dmi
- [ ] icons/obj/syringe.dmi
- [ ] icons/obj/weapons.dmi
- [x] maps/apollo-3.dmm
- [x] maps/apollo-4.dmm
- [x] maps/backup/apollo-1.dmm
- [x] maps/backup/apollo-2.dmm
- [x] maps/exodus-5.dmm

**Comment by lmas** [comment]
Fixed a lot of the code conflicts easily enough, though it looks like you have to fix ALL conflicts in one go before you can merge in any at all. Soo.. it looks like I'm stuck having to fix 'em by myself. Stupid git

**Comment by lmas** [comment]
Alright you plebs, I was somewhat successfull with the merge (see 6422e5582f37d72e8c6a7ba4ee293b15256008f6 in the "bay_merge" branch).

Status:
- **Code**: Fixed without too big problems, I prioritized on bay's changes but made sure to keep our custom modifications.
- **Icons**: Files are fucked. I merged in bay's new icons and let them overwrite our changes, with a few exceptions (some files were easy to fix but I forgot to make a list sooo... yeah, gonna have to inspect them all again). It's really difficult to merge two icon files with +50 sprites in each. I'm going to let someone else fix that mess (someone who might know which sprites changed _wink, wink_).
- **Maps**: No idea why I got merge conflicts on our own maps, but I assumed it was because of the latest changes and decided to merge in the new changes and overwrite my local copies. Exodus map was ignored of course.

Stuff haven't been tested so that's going to be a major pain in the buttocks, though I will get on to that now.
